hazelcast/src/test/java/com/hazelcast/cp/internal/datastructures/lock/FencedLockBasicTest.java

```diff
@@ -103,4 +109,46 @@ public void test_lockFailsAfterCPGroupDestroyed() throws ExecutionException, Int
 
         proxyInstance.getCPSubsystem().getLock(lock.getName());
     }
+
+    @Test(timeout = 60000)
```

Comment:
`assertTrueEventually` has a default timeout, so maybe the test timeout is redundant